probe-dictionary/explorer.css

```diff
@@ -122,3 +122,7 @@ body {
 .hidden {
     display: none;
 }
+
+.nav-item {
+    font-weight: bold;
```

Comment:
How did `text-decoration: underline` look? Users on the web are conditioned to look for underlines (links) and boxes (buttons). ...but if it looks odd, we bold is definitely an improvement over the present style.

Reply:
@chutten I think I prefer without underline... seems cleaner, take a look at the underline below;

![telemetry1after3](https://user-images.githubusercontent.com/29465981/37176767-f7966ff4-22e2-11e8-978e-2056481630c0.PNG)

Let me know if that's something you want me to include.

Reply:
I agree, the underline doesn't work in this design.